stdlib/public/SDK/SceneKit/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ add_swift_library(swiftSceneKit ${SWIFT_SDK_OVERLAY_LIBRARY_BUILD_TYPES} IS_SDK_
   SWIFT_MODULE_DEPENDS_OSX ${SWIFT_SCENEKIT_DEPENDENCIES_NON_WATCHOS} AppKit
   SWIFT_MODULE_DEPENDS_IOS ${SWIFT_SCENEKIT_DEPENDENCIES_NON_WATCHOS} UIKit
   SWIFT_MODULE_DEPENDS_TVOS ${SWIFT_SCENEKIT_DEPENDENCIES_NON_WATCHOS} UIKit
-  SWIFT_MODULE_DEPENDS_WATCHOS Foundation
-  SWIFT_MODULE_DEPENDS simd
+  SWIFT_MODULE_DEPENDS_WATCHOS
+  SWIFT_MODULE_DEPENDS Foundation simd
```

Comment:
You can probably just remove the previous line, but it wasn't incorrect the old way—the non-watchOS platforms already indirectly depend on Foundation.